test/index.js

```diff
@@ -175,14 +183,14 @@ describe('<ReactQuill />', function() {
   it('removes focus from the editor when calling blur()')
 
   /**
-   * In a browser, querySelector('.ql-editor').textContent = 'hi' would 
+   * In a browser, querySelector('.ql-editor').textContent = 'hi' would
    * trigger a 'text-change' event, but here it doesn't. Is the polyfill
    * for MutationObserver not working?
    */
   it('calls onChange after the textContent of the editor changes')
 
   /**
-   * This is hard to do without Selenium's 'type' function, but it is the 
+   * This is hard to do without Selenium's 'type' function, but it is the
```

Comment:
I think a the tests would run in [Cypress](https://cypress.io) fairly well. Might want to try that as a followup refactor.